cloud-scanner-cli/src/boavizta_api_v1.rs

```diff
@@ -357,7 +357,7 @@ mod tests {
     async fn get_api_version() {
         let api: BoaviztaApiV1 = BoaviztaApiV1::new(TEST_API_URL);
         let version = api.get_api_version().await;
-        let expected = Some("1.3.7".to_owned());
+        let expected = Some("1.3.8".to_owned());
```

Comment:
Let's do the version upgrade separately across the whole project. 
https://github.com/Boavizta/cloud-scanner/blob/main/docker-compose.yml#L60 will need changing as well

Reply:
Registered an issue to track this upgrade separately.
https://github.com/Boavizta/cloud-scanner/issues/681